package.json

```diff
@@ -28,13 +28,14 @@
     "prettier:ts": "prettier --write \"src/**/*.ts?(x)\"",
     "prettier:styles": "prettier --write \"src/**/*.scss\"",
     "preversion": "npm ci && npm test",
+    "previewChangelog": "standard-version --skip.commit --skip.tag --silent && cat CHANGELOG.md && git reset --hard --quiet",
     "release": "git push --follow-tags && npm publish && ts-node ./support/githubReleaseFromChangelog.ts",
     "start": "concurrently --kill-others --raw \"npm run demoPageSync\" \"tsc --project ./tsconfig-demos.json --watch\" \"stencil build --dev --watch --serve\"",
     "test": "stencil test --spec --e2e",
     "test:headed": "stencil test --spec --e2e --devtools",
     "test:watch": "stencil test --spec --e2e --watch",
     "test:watchAll": "stencil test --spec --e2e --watchAll",
-    "version": "npm run build && npm run docs && standard-version && git add ."
+    "version": "npm run build && npm run docs && standard-version --skip.bump --skip.commit --skip.tag && git add ."
```

Comment:
suggestion: Maybe spilit `standard-version --skip.bump --skip.commit --skip.tag` into its own command to make this one easier to read.

Reply:
I did a little digging and applied some `yargs` magic to override a top-level skip (there's no example like this in the `standard-version` doc)  to keep this clean.